docs/source/upgrade_to_newest_version.md

```diff
@@ -39,7 +39,7 @@ If you did not vendor the shim in your v1.4 chaincode, the old v1.4 chaincode im
 
 At this point, you have two options:
 
-1. If the entire channel is ready to upgrade chaincode, you can upgrade the chaincode on all peers and on the channel (using either the old or new lifecycle depending on the `Application` capability level you have enabled). The best practice at this point would be to vendor the new Go chaincode shim using modules.
+1. If the entire channel is ready to upgrade chaincode, you can upgrade the chaincode on all peers and on the channel (using either the old or new lifecycle depending on the `Application` capability level you have enabled). The best practice at this point would be to vendor the new Go chaincode shim. If you are already using an existing tool such as ``govendor`` you may continue using it to vendor the new chaincode shim. Best practice, however, would be to use Go modules to vendor the new chaincode shim, as modules are now the de facto standard for dependency management in the Go ecosystem. Since Fabric 2.0.0, chaincode using Go modules without vendored dependencies is also supported.
```

Comment:
I'm good with the technical content, this is exactly what I had in mind, but I think it would be better in the first paragraph of this section to help people get in front of the issue before they upgrade, rather than put it down here which is the section about how to recover if you get into this unfortunate situation after upgrade.

Reply:
Moved it, thank you.